services/web/src/pages/sign-in/index.page.jsx

```diff
@@ -37,25 +38,31 @@ const SignIn = () => {
           onSubmit={handleSubmit(onSubmit)}
           className={styles.form}
         >
-          <Input
-            name="email"
+          <TextInput
+            {...register('email')}
             label="Email Address"
+            labelProps={{
+              'aria-invalid': errors.email && 'true',
```

Comment:
do we really need `&& 'true'` 🤔

Reply:
Yep, cuz it's html attr value — it should be string.